Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -319,11 +321,14 @@ namespace uWS {
             dst[0]    = (std::byte)((flags & SND_NO_FIN ? 0 : 128) | (compressed ? SND_COMPRESSED : 0));
             if ( !(flags & SND_CONTINUATION) ) { dst[0] |= (std::byte)opCode; }
 
-            std::byte mask[4];
+            std::array<std::byte, 4> mask{};
+
             if ( !isServer ) {
                 ((uint8_t*)dst)[1] |= 0x80;
-                uint32_t random = litecore::RandomNumber();
-                memcpy(mask, &random, 4);
+                uint32_t  random = litecore::RandomNumber();
```

Comment:
What's the motivation of this? To avoid endianness?

Reply:
This seems like a side effect of using std::rotate.  This function can only operate on std containers, which means std::array in this case.  Though I'd prefer the use of a slice, and `SecureRandomize` here rather than a memcpy.